<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quick_note::MessageWidget</class>
 <widget class="QWidget" name="quick_note::MessageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>78</width>
    <height>26</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="msg_label">
     <property name="text">
      <string notr="true">TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="edit_btn">
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="icon">
      <iconset resource="../quick_note.qrc">
       <normaloff>:/quick_note/icon.svg</normaloff>:/quick_note/icon.svg</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../quick_note.qrc"/>
 </resources>
 <connections/>
</ui>
